### ISSUE TITLE
Run `git init` after installation process

### ISSUE THREAD

**Issue by secretrobotron** [mozilla-appmaker/component-template#14]
I noticed that I still had to set up my git environment after creating a component with `component-template` following the README. Can we kick of a `git init` at the end? Maybe even commit everything afterward as an `initial commit`?

**Comment by davidascher** [comment]
yeah see the  generated readme for the full set of steps we could automate.
